### PR TITLE
Add passed features documentation

### PR DESCRIPTION
**Pull request by SirOibaf**

### CODE REVIEW

File: docs/user_guides/fs/feature_view/feature-vectors.md [hunk 1]
```diff
@@ -44,6 +44,32 @@ If you have specified transformation functions when creating a feature view, you
 feature_view.init_serving(training_dataset_version=1)
 ```
 
-
 ## Passed features
-Coming soon....
+If some of the features values are only known at prediction time and cannot be computed and cached in the online feature store, you can provide those values as `passed_features` option. The `get_feature_vector` method is going to use the passed values to construct the final feature vector to submit to the model.
+
+You can use the `passed_features` parameter to overwrite individual features being retrieved from the online feature store. You can also use the parameter to provide values for all the features which are part of a specific feature group and used in the feature view. In this second case, you do not have to provide the primary key value for that feature group as no data needs to be retrieved from the online feature store.
```
Comment (kennethmhc) [inline docs/user_guides/fs/feature_view/feature-vectors.md:50]: An example for the second cases would be nice.

File: docs/user_guides/fs/feature_view/feature-vectors.md [hunk 2]
```diff
@@ -44,6 +44,32 @@ If you have specified transformation functions when creating a feature view, you
...
+
+    # get multiple vectors
+    feature_view.get_feature_vectors(
+        entry = [
+            {"pk1": 1, "pk2": 2},
+            {"pk1": 3, "pk2": 4},
+            {"pk1": 5, "pk2": 6}
+        ],
+        passed_features = [
+            {"feature_a": "value_a"},
```
Comment (kennethmhc) [inline docs/user_guides/fs/feature_view/feature-vectors.md:70]: what happen if feature groups have features of same name?

**Review by kennethmhc — CHANGES_REQUESTED** [review]

File: docs/user_guides/fs/feature_view/feature-vectors.md (hunk 2 above)
Comment (SirOibaf) [reply docs/user_guides/fs/feature_view/feature-vectors.md:70]: You should be using the prefix in the feature view.

File: docs/user_guides/fs/feature_view/feature-vectors.md (hunk 1 above)
Comment (SirOibaf) [reply docs/user_guides/fs/feature_view/feature-vectors.md]: Done

**Review by kennethmhc — APPROVED** [review]

**Pull request merged**